cypress/integration/InstancesTest.ts

```diff
@@ -79,177 +85,256 @@ describe("Instances Test", () => {
         .ouiaId(newInstanceName, "PF4/TableRow")
         .should("not.exist");
     });
-  });
 
-  describe("the Instance statuses", () => {
-    beforeEach(() => {
-      cy.visit("/");
-      cy.ouiaId("loading-table", "PF4/Card", { timeout: 30000 }).should(
-        "not.exist"
-      );
-    });
+    it("Instance List pagination test", () => {
+      // cy.ouiaId("OUIA-Generated-Button-plain-3", "PF4/Button").click();       // next button
+      // cy.ouiaId("OUIA-Generated-Button-plain-2", "PF4/Button").click();       // previous button
+      cy.get("[aria-label='Go to first page']").should("be.disabled");
+      cy.get("[aria-label='Go to previous page']").should("be.disabled");
+      cy.get("[aria-label='Go to last page']").should("not.be.disabled");
+      cy.get("[aria-label='Go to next page']").should("not.be.disabled");
+      // Go to next page
+      cy.ouiaId("OUIA-Generated-Button-plain-3", "PF4/Button").click();
```

Comment:
Try to avoid using generated ID. It is fine if you can use `[aria-label='.....']`.

Reply:
Done